site/docs/spark.md

```diff
@@ -286,6 +286,11 @@ val df = spark.read
     .table("prod.db.table")
 ```
 
+!!! Warning
+    When reading with DataFrames in Spark 3, use `table` to load a table by name from a catalog.
+    Using `format("iceberg")` loads an isolated table reference that is not refreshed when other queries update the table.
```

Comment:
@dongjoon-hyun, this is the warning I've added to make people aware of the issues with the `DataFrameReader`. Please take a look if you have time. There is also one below for the v1 `DataFrameWriter` API.

Reply:
Thank you, @rdblue . It looks good to me.